src/lib/common/Kconfig

```diff
@@ -310,6 +310,10 @@ config SOL_BUS
 	default n
 
 menu "Hardware Options"
+config USE_AIO
+	bool "AIO Support"
```

Comment:
I liked the suggested string, but as a side comment, if it was Asynchronous IO it wouldn't be in Hardware Options anyway ;-)